File: 250722/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
using namespace std;

int main() {
    string arr[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    char c;
    int cnt = 0;
    cin >> c;
    for (int i = 0; i < 5; i++) {
        if (arr[i][2] == c || arr[i][3] == c) {
            cout << arr[i] << endl;
            cnt++;
        }
    }
    cout << cnt;
    return 0;
}

File: 250723/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.kt
import java.io.BufferedReader
import java.io.BufferedWriter

fun main() {
    val strArray = arrayOf("apple", "banana", "grape", "blueberry", "orange")
    val c = BufferedReader(System.`in`.reader()).use {
        it.readLine().first()
    }
    val result = strArray.filter { it[2] == c || it[3] == c }
    BufferedWriter(System.out.writer()).use { bw ->
        result.forEach {
            bw.write(it)
            bw.newLine()
        }
        bw.write(result.size.toString())
    }
}